protoc-gen-swagger/genswagger/atlas_patch.go

```diff
@@ -230,6 +230,7 @@ The service-defined string used to identify a page of resources. A null value in
 							op.Responses.StatusCodeResponses[opToStatusCode(on)] = rsp
 							delete(sw.Definitions, trim(rsp.Ref))
 							delete(op.Responses.StatusCodeResponses, 200)
+							delete(op.Responses.StatusCodeResponses, 201)
```

Comment:
Solution fix current problem, but can bring new some time in a the future f.e. someone return 202 or somethis like that. Is it better to make a general solution, just some function which clean up a whole range of success statuses 2** or something like that

Reply:
@burov i have seen this comment as part of atlas-patch "Ability to wrap Responses with correct error codes (200 - for GET, 201 - for POST/PUT/PATCH, 204 - for DELETE)".  So i have removed 201 alone.